src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -1060,6 +1156,19 @@ export default {
 			this.$refs.tabs.focusActiveTabContent()
 		},
 
+		/**
+		 * Check if the toggle button container is available
+		 */
+		checkToggleButtonContainerAvailability() {
+			// Toggle button must be rendered, but there is no element to teleport it to
+			if (this.open === false && !this.noToggle && !this.ncContentSelector) {
+				console.warn(
```

Comment:
why not Vue.warn ?

Reply:
I'd keep it to have warning also in the production build of an app.

But we should definitely depside what we should use in which case for warnings in frontend.